keys.go

```diff
@@ -324,9 +326,16 @@ type Policy struct {
 	CRN       string     `json:"crn,omitempty"`
 	UpdatedAt *time.Time `json:"lastUpdateDate,omitempty"`
 	UpdatedBy string     `json:"updatedBy,omitempty"`
-	Rotation  struct {
-		Interval int `json:"interval_month,omitempty"`
-	} `json:"rotation,omitempty"`
+	Rotation  *Rotation  `json:"rotation,omitempty"`
+	DualAuth  *DualAuth  `json:"dualAuthDelete,omitempty"`
+}
+
+type Rotation struct {
+	Interval int `json:"interval_month,omitempty"`
+}
+
+type DualAuth struct {
+	Enabled *bool `json:"enabled,omitempty"`
```

Comment:
Lets move this stuff to a `policy.go` file to keep the new types clear of the key types